vit-servicing-station-cli/src/csv/loaders.rs

```diff
@@ -0,0 +1,122 @@
+use crate::{db_utils::db_file_exists, task::ExecTask};
+use csv::Trim;
+use serde::de::DeserializeOwned;
+use std::io;
+use structopt::StructOpt;
+use vit_servicing_station_lib::db::{
+    load_db_connection_pool, models::funds::Fund, models::proposals::Proposal,
+    models::voteplans::Voteplan,
+};
+
+#[derive(Debug, PartialEq, StructOpt)]
+pub enum CSVDataCmd {
+    /// Add provided tokens to database. If --tokens is not provided the binary will read them from the `stdin`
+    Dump {
+        /// URL of the vit-servicing-station database to interact with
+        #[structopt(long = "db-url")]
+        db_url: String,
+
+        /// List of tokens in URL safe base64. If --tokens is not provided the binary will read them from the `stdin`
+        #[structopt(long = "funds")]
+        funds: String,
+
+        #[structopt(long = "voteplans")]
+        voteplans: String,
+
+        #[structopt(long = "proposals")]
+        proposals: String,
+    },
+}
+// type Fundd = (String, String, String, String, i64, i64, i64);
+impl CSVDataCmd {
+    fn load_from_csv<T: DeserializeOwned>(csv_path: &str) -> io::Result<Vec<T>> {
+        let mut reader = csv::ReaderBuilder::new()
+            .flexible(true)
+            .has_headers(true)
+            .quoting(true)
+            .quote(b'"')
+            .trim(Trim::All)
+            .from_path(csv_path)?;
+        let mut results = Vec::new();
+        for record in reader.deserialize() {
+            results.push(record?);
+        }
+        Ok(results)
+    }
+
+    fn handle_dump(
+        db_url: &str,
+        funds_path: &str,
+        voteplans_path: &str,
+        proposals_path: &str,
+    ) -> io::Result<()> {
+        db_file_exists(db_url)?;
```

Comment:
Does it make sense to create the db if missing? We could have it create by default if missing or add another bool flag like `--create-db`

Reply:
> Does it make sense to create the db if missing? We could have it create by default if missing or add another bool flag like `--create-db`

If we accept that the flow would be first creating the db from the service itself and then populate the db from the cli, this is not very important right now and may be considered nice to have

Reply:
It should be done with diesel. But I'll check to implement a tiny command for creating the db and run the migrations (coming in another PR).